### PR TITLE
worker: fix postMessage

### PR DESCRIPTION
**Pull request by saghul**
It accepts a single argument.

Fixes: https://github.com/saghul/txiki.js/issues/556

### CODE REVIEW

File: src/js/polyfills/worker.js [hunk 1]
```diff
@@ -26,8 +26,8 @@ class Worker extends EventTarget {
         this[kWorker] = worker;
     }
 
-    postMessage(...args) {
-        this[kWorker].postMessage(args);
+    postMessage(message) {
+        this[kWorker].postMessage(message);
```
[the remark this answers is not in the document]
Comment (saghul) [reply src/js/polyfills/worker.js:30]: It won't fail, the argument will be ignored. You can pass as many parameters as you want.

[the remark this answers is not in the document]
Comment (EmixamPP) [reply src/js/polyfills/worker.js:30]: Let me know if I should create a PR with what I propose or not

[the remark this answers is not in the document]
Comment (EmixamPP) [reply src/js/polyfills/worker.js:30]: > It won't fail, the argument will be ignored. You can pass as many parameters as you want.

Ok fine, good to know!

**Comment by EmixamPP** [comment]
That fix works for me

File: src/js/polyfills/worker.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (saghul) [reply src/js/polyfills/worker.js:30]: > Let me know if I should create a PR with what I propose or not

Adding transfer list support requires structured cloning, which is alas not as straightforward as I thought.

For now this should do, but I'd like have transfer support in the future.

[the remark this answers is not in the document]
Comment (EmixamPP) [reply src/js/polyfills/worker.js:30]: Yeah I won't be able to implement that :sweat_smile:

**Pull request merged**